src/benchmarklib/benchmark_runner.cpp

```diff
@@ -244,19 +247,19 @@ void BenchmarkRunner::_benchmark_individual_queries() {
       if (currently_running_clients.load(std::memory_order_relaxed) < _config.clients) {
         currently_running_clients++;
 
+        const auto pipeline_metrics = std::make_shared<SQLPipelineMetrics>();
+
         // The on_query_done callback will be appended to the last Task of the query,
         // to measure its duration as well as signal that the query was finished
-        const auto query_run_begin = std::chrono::steady_clock::now();
-        auto on_query_done = [query_run_begin, &currently_running_clients, &result, &state]() {
+        auto on_query_done = [/*query_run_begin,*/ pipeline_metrics, &currently_running_clients, &result, &state]() {
```

Comment:
?

Reply:
This comment was marked as resolved, but I can still see `/*query_run_begin*/`. Did anything go wrong here?

Reply:
apparently I didn't push :(